bank_logos.json

```diff
@@ -0,0 +1,311 @@
+[
+    {
+        "name": "Abbey Mortgage Bank",
+        "logo": "https://drive.google.com/file/d/1VIw1PuIf-q2Wfbzqxc5fLvUi1LcDDBoE/view?usp=sharing"
```

Comment:
So this json file needs to have proper structure for the json which is 

```
  {
    "name": "Access Bank",
    "slug": "access-bank",
    "code": "044",
    "ussd": "*901#"
  },
```

Then you need to pass the images to the logos folder.